keycloak/deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# This script is used to deploy the keycloak to Greenstand k8s cluster
+
+# Prompt user to choose the environment
+echo "Please choose the environment to deploy the keycloak"
+echo "1. dev"
+echo "2. test"
+echo "3. prod"
+read -p "Enter your choice: " choice
+deploy_env=$(node -e 'console.log(["dev", "test", "prod"][process.argv[1] - 1])' $choice)
+echo "The environment to deploy is: $(echo ${deploy_env})"
+#TODO
+read -p "Enter any key to continue: " key
+
+# check the k8s cluster
+echo "Checking the k8s cluster"
+current_k8s_cluster=$(kubectl config current-context)
+echo "The current k8s cluster is: ${current_k8s_cluster}"
+#TODO here we use the name in config/context, it might be problematic
+# maybe a cluster configmap is good to have: https://stackoverflow.com/questions/38242062/how-to-get-kubernetes-cluster-name-from-k8s-api
+is_cluster_ok=$(node ./lib/js/checkK8sClusterNameByEnv.js ${deploy_env} ${current_k8s_cluster})
```

Comment:
Can't we use doctl to switch the context for the user?

Reply:
There's a script in this repo that does that

Reply:
Thanks for the reminder